### PR TITLE
Added routes to homepage and about page in the NavBar

### PR DESCRIPTION
**Pull request by denibulkashvili**
changes in this pr 
-added react-router, react-router-dom
-wrapped App inside BrowserRouter in index.js
-created Main component for routes
-added links in the navbar via material-ui buttons
-created dummy about component for testing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,8 @@
     "eslint-plugin-react": "^7.10.0",
     "react": "^16.4.0",
     "react-dom": "^16.4.0",
+    "react-router": "^4.3.1",
+    "react-router-dom": "^4.3.1",
```
Comment (kuromoka) [inline package.json]: according to this discussion, seems right to add only react-router-dom.
https://github.com/ReactTraining/react-router/issues/4648

File: src/components/About.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+
+class About extends Component {
+  render() {
+   return (
+    <div>
+      <h2>About</h2>
```
Comment (kuromoka) [inline src/components/About.jsx]: including other places, could you fix indent? 
I hope you fix along eslint rules as far as possible. :pray:

File: package.json (hunk 1 above)
Comment (denibulkashvili) [reply package.json]: good point, react-router-dom should have everything needed for web.
I will remove react-router

File: src/components/NavBar.jsx [hunk 3]
```diff
@@ -1,15 +1,27 @@
...
+  return(
     <div>
       <AppBar position="static">
-        <Toolbar>
-          <Typography variant="title" color="inherit">
-            YouTube Clone Logo
-          </Typography>
-        </Toolbar>
+        <Typography variant="display1" color="inherit">
+          <Grid container sm={12} justify="center" alignItems="center">
```
Comment (kuromoka) [inline src/components/NavBar.jsx]: this line is returning error as below.
```
Warning: Failed prop type: The property `sm` of `Grid` must be used on `item`.
```
it hasn't shown after I have tried deleting `sm={12}`.

**Pull request merged**